src/io/fetch-stream-loader.js

```diff
@@ -232,6 +232,10 @@ class FetchStreamLoader extends BaseLoader {
                 (this._contentLength !== null && this._receivedLength < this._contentLength))) {
                 type = LoaderErrors.EARLY_EOF;
                 info = {code: -1, msg: 'RangeLoader meet Early-Eof'};
+            } else if (e.message === 'network error' &&
+                (this._contentLength !== null && this._receivedLength > this._contentLength)) {
+                type = LoaderErrors.END_STREAM_NO_INFO;
+                info = {code: -1, msg: 'Stream done without properly message'};
```

Comment:
This looks great, only suggestion is the message here, since the received length is bigger than content length maybe we return
"Stream complete with length larger than content length"

Thoughts?